### PR TITLE
Blog post for Euwe GA announcement

### PR DESCRIPTION
**Pull request by cybette**

### CODE REVIEW

File: site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+[![Container Deployment Wizard](/assets/images/blog/Euwe_GA_container_deployment_wizard_thumb.png)](/assets/images/blog/Euwe_GA_container_deployment_wizard.png)
+
+(more images in the [PR](https://github.com/ManageIQ/manageiq/pull/7620))
+
+Plenty of new reports and metrics enhancements have been included to let you manage containers with confidence. Pods for images per project and Pods per node are added to operations reporting. Additionally, you can get Chargeback rates for container images filtered by image tags, with support for custom attributes in the reports. Speaking of tags, container objects can be auto-tagged with the same labels in Kubernetes/OpenShift.
+
+The status of metrics collection is shown similarly to inventory collection: the success/failure of the last collection is displayed. Now there’s metrics for disconnected container groups as well. Previously we can only obtain the metrics for container groups that were active when metrics were collected (once per hour), so transient groups (that were created and destroyed between collection times) were not recorded. With the new rollout, metrics for such container groups are made available.
+
```
Comment (mfeifer) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: Now, there are metrics...

Comment (mfeifer) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: Previously, we could only obtain....

File: site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Load balancer inventory collection has been added to Azure EMS refresh, as well as support for external proxy. The issue where inventory collection would bypass the proxy when collecting templates has been fixed. You can also make use of provider-specific logging with Azure to make debugging much easier.
+
+A new cache module is deployed in SmartState for Microsoft Azure, which reduces the average read requests from &gt;7000 to &lt;1000. Scan times have also been reduced from &gt;20 min to &lt;5 min!
+
+Finally, we noticed that Azure was missing pre- and post-provision methods in automate modeling, so Azure method points have been added for cloud provisioning in Automate.
+
+[![Azure PrePosition](/assets/images/blog/Euwe_GA_azure_preposition.png)](/assets/images/blog/Euwe_GA_azure_preposition.png)
+
+Many corresponding enhancements are applied to Google Compute Engine, including metrics, external proxy support, load balancer refresh, and pre/post-provisioning methods and schema changes in the method class for Automate.
+
```
Comment (mfeifer) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: Many corresponding enhancements WERE.....

File: site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+[![Google PostPosition](/assets/images/blog/Euwe_GA_google_postposition_thumb.png)](/assets/images/blog/Euwe_GA_google_postposition.png)
+
+During provisioning, the deprecated images are now hidden. Support for parsing Google health checks is added for load balancer refresh.
+
+In GCE, there are [preemptible instances](https://cloud.google.com/compute/docs/instances/preemptible). With Google’s contributions, we have inventory collection and provisioning of these preemptible VMs. Support for instance retirement is improved too.
```
Comment (mfeifer) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: ....improved, too.

File: site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Creating, updating and deleting cloud tenants and host aggregates can now be carried out through the user interface from the Clouds menu under Compute.
+
+[![Add New Cloud Tenant](/assets/images/blog/Euwe_GA_add_cloud_tenant_thumb.png)](/assets/images/blog/Euwe_GA_add_cloud_tenant.png)
+
+Block storage service is introduced with Cinder backup and restore actions. Snapshot functionality has been added for OpenStack cloud volumes and VMs, and you can create and remove snapshots from the UI. New topology view for OpenStack cloud provider shows the availability zones, tenants and instances.
+
+Flavors are mapped to cloud tenants during OpenStack cloud refresh so that they are grouped correctly. Previously Floating IPs can be assigned only during provisioning, however with this release, associating/disassociating Floating IPs can be controlled from the configuration menu after provisioning.
```
Comment (mfeifer) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: Previously, floating IPs could only be assigned during provisioning; however, with this release .....

File: site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+[![Associate Floating IP](/assets/images/blog/Euwe_GA_associate_floating_ip.png)](/assets/images/blog/Euwe_GA_associate_floating_ip.png)
+
+For the OpenStack infrastructure provider, there’s a lot of work done around nodes. The infrastructure nodes can be powered on and off, added or removed. There are additional options for introspecting nodes and setting their provision states. Ironic Controls have been added to the UI, with which Ironic nodes can be registered through Mistral, the OpenStack workflow service. Destroying a node deletes it from Ironic.
```
Comment (cybette) [inline site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md]: I can't remember the source/sprint I got the info from, but it's [introspecting nodes](http://docs.openstack.org/developer/tripleo-docs/advanced_deployment/introspect_single_node.html)

Thanks for all the comments @mfeifer !!

**Pull request merged**